stdlib/2and3/contextlib.pyi

```diff
@@ -59,8 +59,8 @@ if sys.version_info >= (3,):
         def __init__(self) -> None: ...
         def enter_context(self, cm: ContextManager[_T]) -> _T: ...
         def push(self, exit: _CM_EF) -> _CM_EF: ...
-        def callback(self, callback: Callable[..., None],
-                     *args: Any, **kwds: Any) -> Callable[..., None]: ...
+        def callback(self, callback: Callable,
```

Comment:
Can you change None to Any instead of removing the arguments? In general I want to move towards ensuring all generics have type arguments, so we can catch cases where they're accidentally missing.